lib/octokit/client/issues.rb

```diff
@@ -1,367 +1,352 @@
 module Octokit
   class Client
-
     # Methods for the Issues API
     #
     # @see https://developer.github.com/v3/issues/
     module Issues
 
-      # List issues for the authenticated user or repository
+      # List all issues assigned to the authenticated user across all visible repositories including owned repositories, member repositories, and organization repositories
       #
-      # @param repository [Integer, String, Repository, Hash] A GitHub repository.
-      # @param options [Sawyer::Resource] A customizable set of options.
-      # @option options [Integer] :milestone Milestone number.
-      # @option options [String] :state (open) State: <tt>open</tt>, <tt>closed</tt>, or <tt>all</tt>.
-      # @option options [String] :assignee User login.
-      # @option options [String] :creator User login.
-      # @option options [String] :mentioned User login.
-      # @option options [String] :labels List of comma separated Label names. Example: <tt>bug,ui,@high</tt>.
-      # @option options [String] :sort (created) Sort: <tt>created</tt>, <tt>updated</tt>, or <tt>comments</tt>.
-      # @option options [String] :direction (desc) Direction: <tt>asc</tt> or <tt>desc</tt>.
-      # @option options [Integer] :page (1) Page number.
-      # @return [Array<Sawyer::Resource>] A list of issues for a repository.
-      # @see https://developer.github.com/v3/issues/#list-issues-for-a-repository
+      # @option options [String] :filter Indicates which sorts of issues to return. Can be one of:  \* `assigned`: Issues assigned to you  \* `created`: Issues created by you  \* `mentioned`: Issues mentioning you  \* `subscribed`: Issues you're subscribed to updates for  \* `all`: All issues the authenticated user can see, regardless of participation or creation
```

Comment:
This formatting seems weird, is this mean to be a table?

Reply:
Hm that is a bit odd. It looks like it's supposed to be [a list](https://developer.github.com/v3/issues/#parameters). I can see if I can get rid of it 🤔